support/src/testFixtures/java/se/inera/intyg/common/support/facade/testsetup/model/config/ConfigTest.java

```diff
@@ -88,18 +88,22 @@ void shouldIncludeIcon() {
     @Test
     void shouldIncludeConfigText() {
         final var question = getElement();
-        if (question.getConfig().getText() != null) {
+        if (getTextId() != null && !question.getConfig().getText().equals(getTextId())) {
             assertTrue(question.getConfig().getText().trim().length() > 0, "Missing text");
             verify(getTextProviderMock(), atLeastOnce()).get(getTextId());
+        } else {
+            assertEquals(getTextId(), question.getConfig().getText());
         }
     }
 
     @Test
     void shouldIncludeConfigDescription() {
         final var question = getElement();
-        if (getDescriptionId() != null) {
+        if (getDescriptionId() != null && !question.getConfig().getDescription().equals(getDescriptionId())) {
             assertTrue(question.getConfig().getText().trim().length() > 0, "Missing description");
```

Comment:
A misstake when the ConfigTest was introduced originally. Can you change to getConfig().getText()?

Good to run the tests for common before merging just to make sure there aren't any tests that fails. I think the QuestionDiagnosgrundTest didn't fail before because of this error in the testFixture.

Reply:
Done